incubator/moodle/values.yaml

```diff
@@ -0,0 +1,80 @@
+## Bitnami moodle image version
+## ref: https://hub.docker.com/r/bitnami/moodle/tags/
+##
+image: bitnami/moodle
```

Comment:
Looks like you missed this one @jbianquetti-nami. Please specify the image field as `image: bitnami/moodle:3.1.3-r1`